packages/terra-paginator/tests/wdio/paginator-spec.js

```diff
@@ -19,6 +19,13 @@ Terra.describeViewports('Paginator', ['tiny', 'medium', 'large'], () => {
       Terra.validates.element('progressive paginator');
     });
   });
+
+  describe('ProgressivePaginator with Fusion Anchor style', () => {
+    it('default', () => {
```

Comment:
Perhaps update the "default" text with more verbose text?

Reply:
Sure, updated: https://github.com/cerner/terra-core/pull/3900/commits/3fe3671fd8da47f0a3745644ef0cf2210a9bad24#diff-65b614af80147a518e5be5f9679cb2d513196d5503a8cbb0a71f052cbbe10059R24